### DISCUSSION TITLE
not including alot of files

### DISCUSSION THREAD

**Discussion by badatmodding1223** [AssetRipper/AssetRipper#994] [Q&A]
so, im not sure if this is an issue thats why im asking in discussion, but when i try to open the game i ripped, it has a buttload of missing scripts and files in general. does anyone know why? thanks. it only has project settings, assets, packages, and auxiliary files

**Comment by ds5678** [comment]
If it's an Android game there's an open issue about that.

**Reply by badatmodding1223** [reply]
pc, im trying to open blood west, a steam game if that helps

**Reply by marcelherd** [reply]
Which issue would that be? I'm trying to extract a 244 MB _asset_package.asset_ file from [this android game](https://play.google.com/store/apps/details?id=com.ltgames.android.m71.sea) but its not showing up anything in AssetRipper - I would have expected to find all the images in this asset file. Is this still an open issue?

**Reply by ds5678** [reply]
https://github.com/AssetRipper/AssetRipper/issues/307

**Comment by ds5678** [comment]
> missing scripts

I assume you mean the UI message in Unity when you try to view a GameObject?

> files in general

Please elaborate.

**Reply by badatmodding1223** [reply]
a good amount of the textures from the game that were supposed to be ripped from the files are either corrupted or missing, same for every other type of file. half of them are just gone 
![Uploading Screenshot 2023-08-20 154541.png…]()

**Reply by badatmodding1223** [reply]
![Screenshot 2023-08-20 154541](https://github.com/AssetRipper/AssetRipper/assets/142701699/b0bfaee1-2db3-4bfc-8450-b0cd220f3df9)

**Reply by ds5678** [reply]
> corrupted

Show me.

> half of them are just gone 

How do you know they're gone?

**Comment by Zbuffers** [comment]
Hi,
Upon first time opening a decompiled game, you will most probably have some (lots ?) of compile errors.
You need to fix ALL these errors.
Then, when you don't have any compile errors, the scripts will be available on your prefabs.
Happy coding ;)
Z

**Reply by sharky98** [reply]
I did extract a game lately and there were over 500+ compile errors. Most of them just related to some C# features used (or not), `csproj` files wrongly referencing each other and types not found. I was able to get it work mostly by removing a lot of stuff (that I didn't care to be honest, your mileage may vary!).
